src/pages/ContactUs/ContactUs.tsx

```diff
@@ -1,19 +1,42 @@
+import { Helmet } from 'react-helmet-async'
+
 import { ContactUsHero, SectionTitleHero } from './ContactUs.styles'
 import {
   ContentWrapper,
   SectionContactUs,
 } from 'components/ContactUs/ContactUs.styles'
-import ContactUsInformation from 'components/ContactUs/ContactUsInformation/ContactUsInformation'
-import ContactUsForm from 'components/ContactUs/ContactUsForm/ContactUsForm'
 import useIntlMessages from 'hooks/useIntlMessages'
 import H1 from 'ui/Titles/H1'
+import imageLinkPreview from '../../assets/images/image-link-preview.png'
+import { lazy, Suspense } from 'react'
```

Comment:
You need white space here between local and external imports

Reply:
Done!